pkg/service/retrieval/ucan_fx_test.go

```diff
@@ -0,0 +1,129 @@
+package retrieval_test
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"net/http"
+	"testing"
+
+	"github.com/ipfs/go-cid"
+	"github.com/storacha/go-libstoracha/capabilities/space/content"
+	"github.com/storacha/go-libstoracha/testutil"
+	retrievalclient "github.com/storacha/go-ucanto/client/retrieval"
+	"github.com/storacha/go-ucanto/core/delegation"
+	"github.com/storacha/go-ucanto/core/invocation"
+	"github.com/storacha/go-ucanto/core/receipt"
+	"github.com/storacha/go-ucanto/core/result"
+	"github.com/storacha/go-ucanto/ucan"
+	"github.com/stretchr/testify/require"
+	"go.uber.org/fx"
+	"go.uber.org/fx/fxtest"
+
+	"github.com/storacha/piri/pkg/fx/app"
+	piritestutil "github.com/storacha/piri/pkg/internal/testutil"
+	"github.com/storacha/piri/pkg/service/storage"
+	"github.com/storacha/piri/pkg/store/allocationstore/allocation"
+)
+
+func TestFXSpaceContentRetrieve(t *testing.T) {
+	var svc storage.Service
+
+	appConfig := piritestutil.NewTestConfig(t, piritestutil.WithSigner(testutil.Alice))
```

Comment:
I think using `testutil.Service` instead of `testutil.Alice` would avoid some confusion

Reply:
Yes I agree, but I did it this way because the storage tests do the same 😩